### PR TITLE
increase sleep time for monolith oauth2

### PR DESCRIPTION
**Pull request by mshima**
Related to https://github.com/jhipster/generator-jhipster-ionic/pull/712#issuecomment-1236377397

---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/server/templates/src/main/docker/app.yml.ejs [hunk 1]
```diff
@@ -294,7 +302,11 @@ _%>
     # When run with the "prod" Spring profile, it will read the configuration from a Git repository
     # See https://www.jhipster.tech/jhipster-registry/#spring-cloud-config
     environment:
+<%_ if (authenticationTypeOauth2) { _%>
+      - JHIPSTER_SLEEP=40
```
Comment (Tcharl) [inline generators/server/templates/src/main/docker/app.yml.ejs:306]: Should really start to play with health probe and dependency...

Comment (mshima) [reply generators/server/templates/src/main/docker/app.yml.ejs:306]: Can be trick, do you have an idea on how to implement?
It depends on property files, environment variables, profile.

Comment (Tcharl) [reply generators/server/templates/src/main/docker/app.yml.ejs:306]: For keycloak I had something like this: 
```
healthcheck:
      test: ['CMD-SHELL', 'curl -f http://localhost:9080 || exit 1']
      interval: 1m
      timeout: 10s
      retries: 3
```
But you're right, it's case by case approach, the command is different by service and the dependencies as well

And to bootstrap the final class (in CI):
```
startClass="$1"
shift

for var in "$@"
do
    while [ true ]; do
        RUNNING=$(docker inspect --format="{{ .State.Health.Status }}" $var)
        if ["$RUNNING" != "healthy"]; then
            sleep 1
        else
            break
        fi
    done
done

>&2 echo "Container is going to start"
exec java ${JAVA_OPTS} -Djava.security.egd=file:/dev/./urandom -cp /app/resources/:/app/classes/:/app/libs/* "$startClass"
```

Comment (mshima) [reply generators/server/templates/src/main/docker/app.yml.ejs:306]: Would be nice.

**Review by DanielFran — APPROVED** [review]

**Pull request merged**